src/components/FormLayout/FieldGroup.js

```diff
@@ -113,11 +113,8 @@ const FieldGroup = ({
       fields.push(wrapper);
     }
   });
-  return (
-    <React.Fragment>{fields}</React.Fragment>
-  );
-};
-
+  return fields;
```

Comment:
Would it be beneficial for [ `allowedExpressions` ](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-no-useless-fragment.md#allowexpressions) be turned on for this case? since I am assuming that the project will be ported over to typescript eventually? If not at least the rule being there would not remove the indication that this return was a react fragment

Reply:
I'm fine with turning that on or disabling the rule or waiting and adding back fragments where necessary once we do get to typescript. mostly I was trying to add as little extra customization to the eslint config as possible since that makes things easier with many projects